### DISCUSSION TITLE
I want to share the model with all users.

### DISCUSSION THREAD

**Discussion by Taikono-Himazin** [mckaywrigley/chatbot-ui#1844] [Help]
I want to share a model with all users.
When I checked the database with superbase (localhost:54323), I found a column called "sharing" in the models table.
I expect this will allow me to share a model with all users.
However, I don't know what value I should set it to.
Does anyone know?

**Comment by Davitcoin** [comment]
If you want to share a model with all users you can set the API KEY globally, that´s what you´re looking for?
